src/client/mod.rs

```diff
@@ -306,6 +309,17 @@ impl Client {
             .build()
     }
 
+    /// Construct a Hedera client pre-configured for localhost access.
+    #[must_use]
+    pub fn for_localhost() -> Self {
+        let mut network: HashMap<String, AccountId> = HashMap::new();
+        network.insert(DEFAULT_LOCAL_NODE_ADDRESS.to_string(), AccountId::new(0, 0, 3));
+
+        let client = Client::for_network(network).unwrap();
+        client.set_mirror_network([DEFAULT_LOCAL_MIRROR_NODE_ADDRESS.to_string()]);
+        client
+    }
+
```

Comment:
Please don't make this public

Reply:
@izik1 Do you mean only make it accessible to the testing environment?